src/codegen/doc/mod.rs

```diff
@@ -503,7 +513,30 @@ where
             )?;
         }
 
-        for parameter in fn_.parameters() {
+        let parameters: Vec<_> = fn_
+            .parameters()
+            .iter()
+            .filter(|param| {
+                let is_array_length_param = if param.name.len() > 3 {
+                    let array_param_name = &param.name[2..];
+                    &param.name[0..2] == "n_"
+                        && fn_
+                            .parameters()
+                            .iter()
+                            .find(|p| p.name == array_param_name)
+                            .is_some()
+                } else {
+                    false
+                };
+
+                !IGNORED_C_FN_PARAMS.contains(&param.name.as_str())
+                    && param.direction != ParameterDirection::Out
+                    && !(param.name == "error" && param.c_type != "GError**")
```

Comment:
This one is something i'm not super sure about :)

Reply:
Shouldn't this be:

```suggestion
                    && !(param.name == "error" && param.c_type == "GError**")
```